src/runtimes/rego/stack.rs

```diff
@@ -6,10 +6,12 @@ use crate::{
         context_aware, gatekeeper_inventory::GatekeeperInventory, opa_inventory::OpaInventory,
     },
 };
-use anyhow::{anyhow, Result};
+use anyhow::Result;
```

Comment:
Ok, I guess that means we are still going to keep anyhow around, but just for internal errors?